packages/hash/frontend/src/blocks/page/PageTitle.tsx

```diff
@@ -70,16 +94,34 @@ export const PageTitle: FunctionComponent<PageTitleProps> = ({
 
   // TODO: Assign appropriate a11y attributes
   return (
-    <input
-      placeholder="A title for the page"
-      disabled={updateEntityLoading}
-      onChange={handleInputChange}
+    <Typography
```

Comment:
I was not aware of `MUI Textarea Autosize` was a thing while developing this :D I feel like we'll eventually replace the custom page title component with a block inside `ProseMirror` to have seamless editing experience instead of managing the focus between document & title manually. So I propose not spending more time on the current title as it works OK (afaik) for now. What do you think?

Reply:
There is an issue with the current approach when toggling bold (CTRL + B) or italicized (CTRL + I) text inside the title, where the focus changes position to the start of the title. I think it's because we replace innerText completely.

![bug](https://user-images.githubusercontent.com/37453800/186663781-737d5135-67f1-4c26-b7e9-1d7637c25c03.gif)

I've seen this behaviour before in this [PR](https://github.com/hashintel/hash/pull/647#issuecomment-1174965984) and at the time I thought it had more to do with Prosemirror, but I'm starting to think the same thing is happening here.

It's and edge case but it's easily fixable by changing it to a TextField.

Reply:
cc @CiaranMn for technical ideal here.

Reply:
This should be fixed here https://github.com/hashintel/hash/pull/956/commits#:~:text=use%20TextareaAutosize%20on%20PageTitle